x-pack/platform/plugins/shared/ml/public/application/components/anomalies_table/anomaly_details_utils.tsx

```diff
@@ -201,23 +206,25 @@ export const DetailsItems: FC<{
         title: i18n.translate('xpack.ml.anomaliesTable.anomalyDetails.upperBoundsTitle', {
           defaultMessage: 'Upper bound',
         }),
-        description: formatValue(
-          anomaly.source.anomaly_score_explanation?.upper_confidence_bound,
-          source.function,
-          undefined,
-          source
+        description: (
+          <AnomalyValueDisplay
+            value={anomaly.source.anomaly_score_explanation?.upper_confidence_bound}
```

Comment:
@valeriy42 out of interest, why do some anomalies not have the upper / lower bounds set?

<img width="938" alt="Screenshot 2025-03-27 at 12 52 58" src="https://github.com/user-attachments/assets/74f24fe8-f6f0-43d3-8721-ef6da816f668" />